main/src/main.ts

```diff
@@ -182,6 +182,17 @@ export function createWindow(
         width: windowState.bounds.width,
     })
 
+    if (process.platform === "darwin") {
+        const pressAndHold: boolean = systemPreferences.getUserDefault(
+            "ApplePressAndHoldEnabled",
+            "boolean",
+        )
+
+        if (!pressAndHold) {
+            systemPreferences.setUserDefault("ApplePressAndHoldEnabled", "boolean", "true")
```

Comment:
Wow, super cool! Didn't know we could do this. This will alleviate a common pain point on OSX. Nice find @Akin909 💯